exercises/bob/examples/success-standard/src/Bob.hs

```diff
@@ -1,5 +1,6 @@
 module Bob (responseFor) where
-import Data.Char (isSpace, isUpper, isAlpha)
+import Data.Char (isSpace, isUpper, isAlpha, isPunctuation)
+import Safe
```

Comment:
don't forget to add `safe` to ` exercises/bob/examples/success-standard/package.yaml`.